src/hotspot/share/adlc/main.cpp

```diff
@@ -220,6 +220,7 @@ int main(int argc, char *argv[])
   AD.addInclude(AD._CPP_file, "oops/markOop.hpp");
   AD.addInclude(AD._CPP_file, "oops/method.hpp");
   AD.addInclude(AD._CPP_file, "oops/oop.inline.hpp");
+  AD.addInclude(AD._CPP_file, "opto/c2_MacroAssembler.hpp");
```

Comment:
Please remove "AD.addInclude(AD._CPP_file, "asm/macroAssembler.inline.hpp")" in this file.
c2_MacroAssembler.hpp already includes "asm/macroAssembler.inline.hpp".

Reply:
Done.